scripts/exsrw_pre_post_stat.sh

```diff
@@ -8,7 +8,11 @@
 #-----------------------------------------------------------------------
 #
 . ${USHsrw}/source_util_funcs.sh
-source_config_for_task "task_pre_post|task_run_post" ${GLOBAL_VAR_DEFNS_FP}
+for sect in user nco platform workflow nco global verification cpl_aqm_parm \
+  constants fixed_files grid_params \
+  task_pre_post task_run_post ; do
```

Comment:
There is no `task_pre_post` in `config_defaults.yaml`.

```suggestion
  task_run_post ; do
```